ecosystem/sep-0023.md

```diff
@@ -52,6 +53,7 @@ base-32 encoding of the key, are listed here:
 | STRKEY\_PRIVKEY         | 18 << 3      | S          | no    | PK   |
 | STRKEY\_PRE\_AUTH\_TX   | 19 << 3      | T          | no    | Hash |
 | STRKEY\_HASH\_X         | 23 << 3      | X          | no    | Hash |
+| STRKEY\_SIGNED\_PAYLOAD | 15 << 3      | P          | no    | PK   |
```

Comment:
For my own edification: How are these values (`15 << 3` et. al) decided?

Reply:
See https://go.dev/play/p/GhbGoyh-_1O.

These values define the first character in the base 32 strkey. Base 32 encodes 5 bits into every byte/character. So the first 5 bits define the first character. The next 3 bits are used to define the alg. And that makes up the header that is common to all strkeys.

Reply:
Ah, of course. I expected `15 << 3` to be `P` in ascii, but forgot about the base32 aspect of it all.